links/lavnish.json

```diff
@@ -1,5 +1,38 @@
 {
     "Name": "Lavnish Chaudhary",
     "Desc": "I Live and Code",
-    "Avatar": "https://i.imgur.com/3uEyHgdl.png"
-}
\ No newline at end of file
+    "Avatar": "https://instagram.fblr8-1.fna.fbcdn.net/v/t51.2885…4z64GrvjfrvunVgCb_mgGQ&oe=61CE48CE&_nc_sid=7bff83",
```

Comment:
Woops! Please get a right one. Now you know why I don't use "hotlinking" of images.

![image](https://user-images.githubusercontent.com/1830380/147390807-02a384c4-48b7-4a53-bbf2-cd7310bea7c9.png)